src/widget/form/profileform.cpp

```diff
@@ -361,7 +361,12 @@ void ProfileForm::onDeleteClicked()
                 tr("Are you sure you want to delete this profile?", "deletion confirmation text")))
     {
         Nexus& nexus = Nexus::getInstance();
-        nexus.getProfile()->remove();
+
+        if(!nexus.getProfile()->remove())
+        {
+            GUI::showError(tr("Files could not be deleted!"), tr("Some files could not be deleted, please manually remove them."));
```

Comment:
@sudden6 Well, they could check the log but I see your concern. Should the prompt show the names of the file that could not be deleted? Becauses chances are, unless the user manually tampered with the filesystem, either all will delete or none will.